core/templates/dev/head/pages/community-dashboard-page/question-opportunities/question-opportunities.directive.ts

```diff
@@ -166,8 +166,10 @@ angular.module('oppia').directive('questionOpportunities', [
                       return;
                     }
                     QuestionSuggestionService.submitSuggestion($scope.question,
-                      $scope.associatedSkill, opportunity.heading);
+                      $scope.associatedSkill, opportunity.subheading);
                     $uibModalInstance.close();
+                    AlertsService.addSuccessMessage(
+                      'Submitted question suggestion.');
```

Comment:
Ditto as above move this to the successHandler of the http request.

Reply:
Done.